lib/evmone/analysis.cpp

```diff
@@ -147,18 +152,6 @@ code_analysis analyze(
             break;
         }
 
-        case ANY_DUP:
-            // TODO: This is not needed, but we keep it
```

Comment:
Why is this not needed anymore?

Reply:
The information provided here is not needed because now we have individual functions for each DUP instruction. This is slightly faster. Think of it as this information is encoded in the function pointer. This will also save us space as we only need single function pointer per DUP instruction instead of function pointer + an immediate value.